framework/packages/abstract-core/src/core/manager.rs

```diff
@@ -145,6 +145,19 @@ pub enum InternalConfigAction {
     },
 }
 
+#[cosmwasm_schema::cw_serde]
+#[non_exhaustive]
+pub enum SubAccountAction {
```

Comment:
I thought we could also move the account creation here?

Reply:
Maybe, but that's not the action of sub-account, it's action of this manager